web/src/OverviewItemView.tsx

```diff
@@ -164,6 +167,13 @@ export let OverviewItemBox = styled.div`
     animation: ${barberpole} 8s linear infinite;
   }
 `
+let OverviewItemActions = styled.div`
```

Comment:
are you sure this does the layout correctly when the resource name is long?

usually mixing absolute positioning with flexbox layouts defeats the point of using flexbox, because now the flex doesn't work correctly.

Reply:
Yes, I agree that flexbox layout is better. Fixed!